lib/atomic/activities.ex

```diff
@@ -200,4 +201,146 @@ defmodule Atomic.Activities do
   def change_session(%Session{} = session, attrs \\ %{}) do
     Session.changeset(session, attrs)
   end
+
+  alias Atomic.Activities.Enrollment
+
+  @doc """
+  Returns the list of enrollments.
+
+  ## Examples
+
+      iex> list_enrollments()
+      [%Enrollment{}, ...]
+
+  """
+  def list_enrollments do
+    Repo.all(Enrollment)
+  end
+
+  @doc """
+  Gets a single enrollment.
+
+  Raises `Ecto.NoResultsError` if the Enrollment does not exist.
+
+  ## Examples
+
+      iex> get_enrollment!(123)
+      %Enrollment{}
+
+      iex> get_enrollment!(456)
+      ** (Ecto.NoResultsError)
+
+  """
+  def get_enrollment!(id), do: Repo.get!(Enrollment, id)
+
+  @doc """
+  Creates a enrollment.
```

Comment:
```suggestion
  Creates an enrollment.
```

Same for update and delete